.github/workflows/rust.yml

```diff
@@ -60,6 +60,8 @@ jobs:
         rustup override set ${{ env.rust_min }}
     - name: Print Rust version
       run: rustc --version
+    - name: Use MSRV Cargo.lock
+      run: mv Cargo.lock.msrv Cargo.lock
```

Comment:
One more request: could you add the [`--frozen`](https://doc.rust-lang.org/cargo/commands/cargo-check.html#manifest-options) flag to the invocations of `cargo check` in the `min_rust` job? This should assert that cargo doesn't modify the lockfile due to some reason not accounted by us.